# file: C:\Users\rudra\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\schemathesis\config\_checks.py
# hypothesis_version: 6.135.14

['1', '2', '2xx', '3', '3xx', '4', '400', '401', '403', '404', '406', '422', '428', '4xx', '5', '5xx', 'X', 'XX', '_', '_unknown', 'all', 'enabled', 'expected-statuses', 'expected_statuses', 'ignored_auth', 'limit', 'max_response_time', 'not_a_server_error', 'unsupported_method', 'use_after_free']